components/var_dumper/advanced.rst

```diff
@@ -240,3 +254,51 @@ properties not in the class declaration).
 .. tip::
 
     Before writing your own casters, you should check the existing ones.
+
+Adding semantics with metadata
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.2
+
```

Comment:
Please remove this empty line (the versionadded directives acts differently from the others)